js/components/Block.js

```
@@ -0,0 +1,211 @@
+
+		if (this.element && !this.element.parentNode) {
+			parent.appendChild(this.element);
+			this.isInDOM = true;
+		}
+	}
+
+	/**
+	 * Remove this block from DOM (optimized)
+	 */
+	removeFromDOM() {
+		if (this.element && this.element.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)